[2024-07-09T12:35:45.935+0000] {processor.py:161} INFO - Started process (PID=172) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:35:45.935+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:35:45.936+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:35:45.936+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:35:45.941+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:35:45.939+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/first_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/first_dag.py", line 12, in <module>
    dag = DAG(
          ^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 519, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'First dag' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-07-09T12:35:45.941+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:35:45.958+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.028 seconds
[2024-07-09T12:36:16.125+0000] {processor.py:161} INFO - Started process (PID=184) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:36:16.126+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:36:16.129+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:36:16.128+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:36:16.136+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:36:16.132+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/first_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/first_dag.py", line 12, in <module>
    dag = DAG(
          ^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 519, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'First dag' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-07-09T12:36:16.137+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:36:16.156+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.042 seconds
[2024-07-09T12:36:46.310+0000] {processor.py:161} INFO - Started process (PID=194) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:36:46.310+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:36:46.311+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:36:46.311+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:36:46.315+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:36:46.313+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/first_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/first_dag.py", line 12, in <module>
    dag = DAG(
          ^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 519, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'First dag' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-07-09T12:36:46.315+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:36:46.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.023 seconds
[2024-07-09T12:37:16.517+0000] {processor.py:161} INFO - Started process (PID=205) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:37:16.518+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:37:16.519+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:37:16.519+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:37:16.522+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:37:16.521+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/first_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/first_dag.py", line 12, in <module>
    dag = DAG(
          ^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 519, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'First dag' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-07-09T12:37:16.523+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:37:16.537+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.024 seconds
[2024-07-09T12:37:46.703+0000] {processor.py:161} INFO - Started process (PID=216) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:37:46.703+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:37:46.705+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:37:46.705+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:37:46.708+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:37:46.706+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/first_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/first_dag.py", line 12, in <module>
    dag = DAG(
          ^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 519, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'First dag' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-07-09T12:37:46.708+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:37:46.721+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.023 seconds
[2024-07-09T12:38:09.857+0000] {processor.py:161} INFO - Started process (PID=227) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:09.859+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:38:09.862+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:09.862+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:09.879+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:09.869+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/first_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/first_dag.py", line 31, in <module>
    t1 = PythonOperator(
         ^^^^^^^^^^^^^^
NameError: name 'PythonOperator' is not defined
[2024-07-09T12:38:09.885+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:09.899+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.056 seconds
[2024-07-09T12:38:29.952+0000] {processor.py:161} INFO - Started process (PID=238) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:29.953+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:38:29.954+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:29.954+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:29.968+0000] {processor.py:840} INFO - DAG(s) 'elt_and_dbt' retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:30.158+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:30.157+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:elt_and_dbt
[2024-07-09T12:38:30.165+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:30.164+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:elt_and_dbt
[2024-07-09T12:38:30.170+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:30.170+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:elt_and_dbt
[2024-07-09T12:38:30.171+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:30.171+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-09T12:38:30.179+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:30.179+0000] {dag.py:3118} INFO - Creating ORM DAG for elt_and_dbt
[2024-07-09T12:38:30.187+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:30.186+0000] {dag.py:3954} INFO - Setting next_dagrun for elt_and_dbt to 2024-07-08 00:00:00+00:00, run_after=2024-07-09 00:00:00+00:00
[2024-07-09T12:38:30.198+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.250 seconds
[2024-07-09T12:38:34.042+0000] {processor.py:161} INFO - Started process (PID=243) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:34.043+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:38:34.045+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:34.045+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:34.055+0000] {processor.py:840} INFO - DAG(s) 'elt_and_dbt' retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:34.064+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:34.064+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-09T12:38:34.082+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:34.082+0000] {dag.py:3954} INFO - Setting next_dagrun for elt_and_dbt to 2024-07-08 00:00:00+00:00, run_after=2024-07-09 00:00:00+00:00
[2024-07-09T12:38:34.097+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.064 seconds
[2024-07-09T12:38:38.065+0000] {processor.py:161} INFO - Started process (PID=248) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:38.066+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:38:38.068+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:38.067+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:38.075+0000] {processor.py:840} INFO - DAG(s) 'elt_and_dbt' retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:38.084+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:38.083+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-09T12:38:38.102+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:38.102+0000] {dag.py:3954} INFO - Setting next_dagrun for elt_and_dbt to 2024-07-08 00:00:00+00:00, run_after=2024-07-09 00:00:00+00:00
[2024-07-09T12:38:38.117+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.057 seconds
[2024-07-09T12:38:43.093+0000] {processor.py:161} INFO - Started process (PID=253) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:43.093+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:38:43.095+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:43.094+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:43.098+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:43.096+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/first_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/first_dag.py", line 15, in <module>
    dag = DAG(
          ^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 519, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'first task' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-07-09T12:38:43.098+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:43.113+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.024 seconds
[2024-07-09T12:38:58.197+0000] {processor.py:161} INFO - Started process (PID=264) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:58.197+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:38:58.199+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:58.198+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:58.205+0000] {processor.py:840} INFO - DAG(s) 'first_task' retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:38:58.297+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:58.297+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:first_task
[2024-07-09T12:38:58.306+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:58.305+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:first_task
[2024-07-09T12:38:58.317+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:58.316+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:first_task
[2024-07-09T12:38:58.317+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:58.317+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-09T12:38:58.330+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:58.330+0000] {dag.py:3118} INFO - Creating ORM DAG for first_task
[2024-07-09T12:38:58.339+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:38:58.339+0000] {dag.py:3954} INFO - Setting next_dagrun for first_task to 2024-07-08 00:00:00+00:00, run_after=2024-07-09 00:00:00+00:00
[2024-07-09T12:38:58.350+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.158 seconds
[2024-07-09T12:39:05.224+0000] {processor.py:161} INFO - Started process (PID=269) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:39:05.225+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:39:05.226+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:39:05.226+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:39:05.233+0000] {processor.py:840} INFO - DAG(s) 'first_task' retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:39:05.243+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:39:05.242+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-09T12:39:05.261+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:39:05.260+0000] {dag.py:3954} INFO - Setting next_dagrun for first_task to 2024-07-08 00:00:00+00:00, run_after=2024-07-09 00:00:00+00:00
[2024-07-09T12:39:05.275+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.056 seconds
[2024-07-09T12:39:14.316+0000] {processor.py:161} INFO - Started process (PID=274) to work on /opt/airflow/dags/first_dag.py
[2024-07-09T12:39:14.318+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/first_dag.py for tasks to queue
[2024-07-09T12:39:14.321+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:39:14.320+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/first_dag.py
[2024-07-09T12:39:14.333+0000] {processor.py:840} INFO - DAG(s) 'first_task' retrieved from /opt/airflow/dags/first_dag.py
[2024-07-09T12:39:14.343+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:39:14.343+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-09T12:39:14.362+0000] {logging_mixin.py:188} INFO - [2024-07-09T12:39:14.362+0000] {dag.py:3954} INFO - Setting next_dagrun for first_task to 2024-07-08 00:00:00+00:00, run_after=2024-07-09 00:00:00+00:00
[2024-07-09T12:39:14.376+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/first_dag.py took 0.078 seconds
